src/lsp/cobol_parser/grammar_tokens.mly

```diff
@@ -106,7 +106,7 @@
 %token ATTRIBUTE                    [@keyword]                [@contexts set_stmt]
 %token ATTRIBUTES                   [@keyword]                [@contexts ]
 %token AUTHOR                       [@keyword]
-%token AUTO                         [@keyword]                [@contexts screen_descr_entry]
+%token AUTO                         [@keyword "AUTO", "AUTO-SKIP"] [@contexts screen_descr_entry, accept_stmt]
```

Comment:
`AUTO-SKIP` should not be included here: it's typically reserved on a dialect-specific basis (*e.g,* `mf.words`).

Same for the tokens below (`BEEP`, `FULL`, etc).

Reply:
As a general rule, anything that's not in `reserved.c` from GnuCOBOL should not appear here.